data/showcase/revenu_de_base.yml

```
@@ -4,3 +4,6 @@ description:
   en: Publication of the French Movement for a Basic Income that uses OpenFisca and its Python API to evaluate the cost and benefits of a reform introducing a basic income in France.
 country: FR
 website: https://www.revenudebase.info/2017/04/07/apprehender-cout-dun-revenu-de-base/
+author:
+  type: ngo
+  name: MFRB
```

Are we limited in the number of characters for the layout? If so, please make sure this is reflected in #121  🙂 
```suggestion
  name: Mouvement Français pour un Revenu de Base
```

---

I have updated the authors' names without character restrictions.